vignettes/advs.Rmd

```diff
@@ -0,0 +1,491 @@
+---
+title: "Creating a Pediatrics ADVS ADaM"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating a Pediatrics ADVS ADaM}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiraldev)
+```
+
+# Introduction
+
+This article describes creating a vital signs ADaM for pediatric
+clinical trials. The main focus is around Anthropometric indicators
+(i.e. growth development chart) parameter derivations.
+
+We advise you first consult the `{admiral}` [Creating a BDS Finding ADaM
```

Comment:
Not sure why but it appears as 2 different links.
![image](https://github.com/pharmaverse/admiralpeds/assets/157114584/151e1567-a34e-4636-8fa8-e0f4fc839c33)

Reply:
Same occurs in admiralonco as below so will leave it. Think the first links to the admiral homepage in case people are completely new to the package and the other directly to the vignette:

![image](https://github.com/pharmaverse/admiralpeds/assets/82581364/462c9bc5-73af-48fe-b640-b0783edafc45)

Reply:
ok, let's keep it as in admiralonco.